lib/policy_machine.rb

```diff
@@ -195,6 +195,20 @@ def scoped_privileges(user_or_attribute, object_or_attribute, options = {})
   end
 
   ##
+  # Search for and iterate over a collection in batches
+  def batch_find(type:, query: {}, config: {}, &blk)
+    if policy_machine_storage_adapter.respond_to?(:batch_find)
+      policy_machine_storage_adapter.batch_find(type, query, config, &blk)
+    else
+      method(type.to_s.pluralize).call.select do |pm_el|
```

Comment:
Oh rad. `call(query)` work because of [this](https://github.com/mdsol/the_policy_machine/blob/b9e6dec8bf6da3e20bc9d08471f72ee1472dd6b0/lib/policy_machine.rb#L261) line?